corehq/apps/hqadmin/service_checks.py

```diff
@@ -31,7 +32,12 @@
 from corehq.util.decorators import change_log_level
 from corehq.apps.hqadmin.utils import parse_celery_workers, parse_celery_pings
 
-ServiceStatus = namedtuple("ServiceStatus", "success msg")
+
+@attr.s
```

Comment:
New way of doing classes. Are you hoping to push out this style more broadly in HQ?